packages/base/src/3dview/mainview.tsx

```diff
@@ -619,9 +621,27 @@ export class MainView extends React.Component<IProps, IStates> {
             this._selectedMeshes.push(el as any as BasicMesh);
             el.material.color = SELECTED_MESH_COLOR;
             el.material.linewidth = SELECTED_LINEWIDTH;
+          } else {
+            // Apply objColor for non-selected meshes
+            if (objColor && el.material?.color) {
+              const newColor = new THREE.Color(objColor);
+              el.material.color = newColor;
+              el.userData.originalColor = newColor.clone();
```

Comment:
I feel like we should save `userData.originalColor` in all cases (whether it's selected or not)?

Reply:
Sure, on it.